src/internal/tester/its/resourceService.go

```diff
@@ -0,0 +1,25 @@
+package its
+
+import "github.com/alcionai/corso/src/pkg/path"
+
+type ResourceServicer interface {
+	Resource() IDs
+	Service() path.ServiceType
+}
+
+type resourceAndService struct {
+	ProtectedResource IDs
+	ServiceType       path.ServiceType
+}
+
+func (ras resourceAndService) Resource() IDs {
+	return ras.ProtectedResource
+}
+
+func (ras resourceAndService) Service() path.ServiceType {
+	return ras.ServiceType
+}
+
+func NewResourceService(r IDs, s path.ServiceType) ResourceServicer {
+	return &resourceAndService{r, s}
```

Comment:
nit: please use struct field names. That way if more things are added to the struct the dev making that change doesn't have to find all the initialization sites and add a lot of stuff to them

```suggestion
	return &resourceAndService{
		protectedResource: r,
		serviceType: s,
	}
```